src/chi/components/buttons/buttons.scss

```diff
@@ -484,46 +475,24 @@ $sizes: (
           &.-focus {
             background-color: white;
             border: $border solid rgba(set-color(black), 0.2);
-            box-shadow: 0 0 0 3px rgba(set-color($color, $tone-focus-shadow), 0.4);
           }
         }
 
         .chi-icon {
-          color: $text-color;
+          color: $text-black;
         }
 
         &.-icon {
-          & .chi-icon {
-            color: $text-color;
+          .chi-icon {
+            color: $text-black;
           }
 
+          &:active,
+          &.-active,
           &:hover,
-          &.-hover {
+          &.-hover, {
```

Comment:
@mattnickles is this comma necessary?

Reply:
The entire `-icon` rule wasn't necessary as we no longer specify custom styles for flat light icon buttons. Removed - thanks!